config/gitleaks.toml

```diff
@@ -2077,6 +2077,17 @@ regexTarget = "line"
 regexes = [
     '''--mount=type=secret,''',
 ]
+[[rules.allowlists]]
+condition = "AND"
+paths = [
+    '''\.bb$''','''\.bbappend$''','''\.bbclass$''','''\.inc$''',
```

Comment:
outside of the scope for this PR but it would be nice if this formatting matched the other toml arrays where each line has an entry